modules/govuk_elasticsearch/manifests/init.pp

```diff
@@ -106,13 +121,18 @@
   }
 
   elasticsearch::instance { $::fqdn:
+    notify        => Exec['secure_es_yaml'],
     config        => $instance_config_real,
     datadir       => '/mnt/elasticsearch',
     init_defaults => {
       'ES_HEAP_SIZE' => $heap_size,
     },
   }
 
+  exec { 'secure_es_yaml':
+    command => "chmod 0640 /etc/elasticsearch/${::fqdn}/elasticsearch.yml",
+  }
```

Comment:
Is this the only way to can ensure permissions on the config file? Seems untidy to me, is there a way we can set a default file attribute for that user?